mindjunkies/courses/models.py

```
@@ -72,19 +73,28 @@ class Course(BaseModel):
-
     def __str__(self):
         return self.title
 
     def save(self, *args, **kwargs):
         if not self.slug:
-            self.slug = slugify(self.title)
+            base_slug = slugify(self.title)
+            slug = base_slug
+            # Check if slug already exists
+            ModelClass = self.__class__
+            counter = 1
```

Ruff (0.8.2)

Local variable `counter` is assigned to but never used

Remove assignment to unused variable `counter`

(F841)